src/bigquery_query.ts

```diff
@@ -392,19 +392,19 @@ export default class BigQueryQuery {
       const partitionedField = this.target.partitionedField ? this.target.partitionedField : '_PARTITIONTIME';
       if (this.target.timeColumn !== partitionedField) {
         if (this.templateSrv.timeRange && this.templateSrv.timeRange.from) {
-          const from = `${partitionedField} >= '${BigQueryQuery.formatDateToString(
+          const from = `${partitionedField} >= DATE('${BigQueryQuery.formatDateToString(
```

Comment:
Hi @seuf,
So im checking your update locally and Im running into this BigQuery error:
`No matching signature for operator >= for argument types: TIMESTAMP, DATE. Supported signature: ANY >= ANY at`
Where originally (before the change) my query seemed to work as expected... 
Its probably a missmatch of `DATE` function and `>=` operator. WIll `TIMESTAMP` result with the same expected behaviour? Works for me.. am I missing something?

```suggestion
          const from = `${partitionedField} >= TIMESTAMP('${BigQueryQuery.formatDateToString(
```




My query:

> "#standardSQL
SELECT
 `startExecutionTime` AS time,
  `executionType` AS metric,
  `actualTotalBytesProcessed`
FROM `mydata-1470162410749.EVALUEX_DEV.superQueryAudit`
WHERE
  `startExecutionTime` > TIMESTAMP_MILLIS (1563619227095)  AND
  _PARTITIONTIME >= TIMESTAMP('2019-07-20 13:40:24') AND
  _PARTITIONTIME < TIMESTAMP('2021-07-20 13:40:24')
GROUP BY 1,2,3 
ORDER BY 1,2 LIMIT 1406"

Reply:
Hi @ofir5300 Maybe it depends on the type of the partitioned column. In my case the partioned column type is `DATE`.

Reply:
Is your partition column is your time column?
If so, please try the code below and let me know if it solves it for you
```suggestion
          const from = `${partitionedField} >= ${this.target.timeColumnType}('${BigQueryQuery.formatDateToString(
            this.templateSrv.timeRange.from._d,
            '-',
            true
          )}')`;
```

*Note the closing parentheses in the end

Reply:
If my time column is the partitioned column, the where clause is not added and it works.
But when my time column is another column, the where clause is added and it fail.
I've tried with `${this.target.timeColumnType}`
and I have an error 
```
invalidQuery: No matching signature for operator >= for argument types: DATE, TIMESTAMP. Supported signature: ANY >= ANY at [10:3]"
``̀
When I force the `DATE` cast it works

Reply:
This is because my time column type is `TIMESTAMP` but my partitioned colmun type is `DATE`.
We should fetch the type of the partitioned column.

Reply:
Yes I see and agree with you,
- My example worked with `TIMESTAMP(` and with `${this.target.timeColumnType}(` because it automatically detected `_PARTITIONTIME` as the partition, so on both cases it parsed successfuly as timestamp
- In your example it works with `DATE(` because this was the type of your partitioned field

The thing is I cant at the moment figure out how can we detect the partition field type in cases such as yours

Any suggestions?
Thats what `this.target` object contains:
<img width="497" alt="Screen Shot 2021-08-22 at 16 48 18" src="https://user-images.githubusercontent.com/40420259/130357560-0985e86b-5d8d-4498-bfdc-8ef302ba4e52.png">

Reply:
The only thing I can think of right now is - getting this data from bigquery (few requests are needed)
And I would like to avoid it on the given scope (BigQueryQuery.buildWhereClause)